src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -1067,7 +1067,7 @@ base::Expression opBuilderHelperAppendSplitString(const std::any& definition)
     if (parameters[1].m_value.size() != 1)
     {
         throw std::runtime_error(
-            fmt::format("\"{}\" function: Separator \"{}\" should be one character long",
+            fmt::format(R"("{}" function: Separator "{}" should be one character long)",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings